build.sbt

```diff
@@ -51,6 +51,29 @@ lazy val cluster = project
     libraryDependencies ++= Dependencies.akkaCluster
   ).configs(MultiJvm)
 
+lazy val stressBackend = project.in(file("./stress/backend"))
+  .aggregate(core, cluster)
+  .dependsOn(core, cluster)
+  .settings(moduleName := "kanaloa-stress-backend")
+  .settings(noPublishing:_*)
+  .settings(Dependencies.stressTestDeps:_*)
+
+lazy val stressFrontend = project.in(file("./stress/frontend"))
+  .aggregate(stressBackend)
+  .dependsOn(stressBackend)
+  .settings(moduleName := "kanaloa-stress-frontend")
+  .settings(noPublishing:_*)
+  .settings(Dependencies.stressTestDeps:_*)
+
+lazy val stressGatling = project.in(file("./stress/gatling"))
+  .enablePlugins(GatlingPlugin)
+  .aggregate(stressFrontend)
```

Comment:
being that Gatling only cares about an http host, it should not have any deps on anything, I would argue.  It is just making http requests, and receiving http responses, so why does it need to depend on anything else other than..well, http?